core/src/test/scala/com/salesforce/op/stages/impl/preparators/BadFeatureZooTest.scala

```diff
@@ -541,7 +540,7 @@ class BadFeatureZooTest extends FlatSpec with TestSparkContext with Logging {
     val retrieved = SanityCheckerSummary.fromMetadata(summary.getSummaryMetadata())
 
     // Check that all of the hashed text columns (and the null indicator column itself) are thrown away
```

Comment:
Can you change the comments to agree with the new behavior too? The text field is detected as categorical and pivoted now instead of being hashed.

Reply:
fixed! thanks!